scripts/logid-generator.py

```diff
@@ -18,7 +18,7 @@
 BUILD = 32
 
 # Find log messages, there is a format ID string parameter
-log_re_string = r'log_(?P<TYPE>.*)\s*\(\s*(?P<COMPONENT>[A-Z]*)\s*,\s*(?P<ID>\d+)\s*,\s*(?P<MSG>(.|\s)*?)\s*\)\s*;'
+log_re_string = r'log_(?P<TYPE>.*)\s*\(\s*(?P<COMPONENT>[a-zA-Z_]*)\s*,\s*(?P<ID>\d+)\s*,\s*(?P<MSG>(.|\s)*?)\s*\)\s*;'
```

Comment:
To make it work with the actual component name, e.g.: "A.b.C":
```
log_re_string = r'log_(?P<TYPE>.*)\s*\(\s*(?P<COMPONENT>[a-zA-Z_\.\"]*)\s*,\s*(?P<ID>\d+)\s*,\s*(?P<MSG>(.|\s)*?)\s*\)\s*;'
```

https://regex101.com/r/6ih665/1